packages/rn-tester-e2e/jest.config.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+module.exports = {
+  testTimeout: 60000,
+  bail: 0,
+  setupFilesAfterEnv: ['./jest.setup.js'],
```

Comment:
> A list of paths to modules that run some code to configure or set up the testing framework before each test file in the suite is executed.

Are we wanting to teardown/setup the app for every spec, or do we want to do this once?

Reply:
Hi @NickGerleman,

We do not want to have flakiness on a high level. We want to maintain a high level of quality and truthfulness.

Currently, we leave these steps before each test as it is.

Reply:
> teardown/setup the app for every spec

yeah that's the better way to reduce flakyness.